examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ClusterRoleBinding
```

Comment:
ok i added a bunch more stuff, mostly for auth delegation (which is now enabled, the SC should no longer be directly accessible w/o auth).